app/Nova/Metrics/ActiveAttendanceBreakdown.php

```diff
@@ -81,6 +113,7 @@ public function calculate(Request $request): PartitionResult
      */
     public function uriKey(): string
     {
-        return 'active-attendance-breakdown';
+        return $this->resourceId === -1 ? 'active-attendance-breakdown' :
+            'active-attendance-breakdown-'.$this->attendableType.'-'.$this->resourceId;
```

Comment:
Playing around with this locally and I think this is a little bit cleaner
```suggestion
        return $this->resourceId === -1 ? 'active-attendance-breakdown' :
            '../'.$this->attendableType.'/'.$this->resourceId.'/metrics/active-attendance-breakdown';
```

That way you don't need the card-specific routing logic on the dashboard level, because the URI actually points to the event-specific metric.

Reply:
Unable to make this work - Will try after a different card, and if does not work, will notify

Reply:
They generally would have an 's' appended though I think there's a more robust way to do it. I would just append it for now.

There are only two types of attendables, events and teams, so that should be fine.

Reply:
confirmed, this works - also the uri actually starts with nova-api, not nova/dashboards, for the record